test/helper/stubs.ts

```diff
@@ -0,0 +1,61 @@
+import * as sinon from 'sinon';
+import { commands, window, workspace } from 'vscode';
+
+export function stubExecuteCommand(returns?: any): sinon.SinonStub {
+    return stubObject(commands, 'executeCommand', returns);
+}
+
+export function restoreExecuteCommand() {
+    restoreObject(commands.executeCommand);
+}
+
+export function stubGetConfiguration(returns?: any): sinon.SinonStub {
+    return stubObject(workspace, 'getConfiguration', returns);
+}
+
+export function restoreGetConfiguration() {
+    restoreObject(workspace.getConfiguration);
+}
+
+export function stubInputBox(returns?: any): sinon.SinonStub {
```

Comment:
where is the verb in these function names?

Reply:
verb added